tests/testthat/test-tokens_compound.R

```diff
@@ -129,42 +129,6 @@ test_that("tokens_compound works as expected with nested and overlapping tokens"
     )
 })
 
-test_that("tokens_compound works as expected with collocations", {
```

Comment:
Better not delete. We can test `tokens_compound()` with a save collocation object.

Reply:
Agreed. Added back in 9e9aa94. I put them into a separate test file called `test-collocations.R` to make this easily distinct. (This had been in **quanteda.textstats** but I moved it out.)